README.md

```diff
@@ -27,7 +27,7 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
   * [Solr 4.10.3](http://lucene.apache.org/solr/)
     * Available at: [http://localhost:8080/solr](http://localhost:8080/solr), for indexing & searching your content.
     * Installed in `/var/lib/tomcat7/solr`
-  * [Apache Karaf](http://karaf.apache.org/)
+  * [Apache Karaf 4.0.1](http://karaf.apache.org/)
```

Comment:
Tagging the fcrepo4-vagrant-base-box with versions that align with what has happened in atlas would be helpful. Thanks, @ruebot!